WcaOnRails/app/webpacker/components/Panel/Leader/GroupsManager.jsx

```diff
@@ -98,6 +102,40 @@ function GroupTable({ groupId }) {
           ))}
         </Table.Body>
       </Table>
+      <Modal
+        size="fullscreen"
+        onClose={() => {
+          setModelOpen(false);
+        }}
+        open={modalOpen}
+      >
+        <Modal.Content>
+          <Form>
+            <Form.Field
+              label="New Member"
+              control={WcaSearch}
+              value={newMemberUser}
+              onChange={(_, { value }) => setNewMemberUser(value)}
+              model="user"
+              multiple={false}
+            />
+            <Form.Button onClick={() => setModelOpen(false)}>Cancel</Form.Button>
+            <Form.Button
+              disabled={!newMemberUser}
+              onClick={() => {
+                setModelOpen(false);
+                save(apiV0Urls.userRoles.create(), {
+                  userId: newMemberUser.id,
+                  groupId: group.id,
+                  status: statusObjectOfGroupType(group.group_type).member,
+                }, () => sync(), { method: 'POST' });
```

Comment:
You can pass in the functional reference to `sync` directly: `save(url, body, sync, { method: 'POST' })`

Reply:
Ouch, that was a simple mistake. I don't know how I missed it. Changed it now.